app/src/androidTest/java/com/github/se/project/ui/lesson/AddLessonTest.kt

```diff
@@ -128,11 +146,11 @@ class AddLessonTest {
 
     // Select Date and Time
     composeTestRule.onNodeWithTag("DateButton").performClick()
-    composeTestRule.waitUntil(15000) { composeTestRule.onNodeWithText("OK").isDisplayed() }
-    composeTestRule.onNodeWithText("OK").performClick()
+    composeTestRule.waitUntil(15000) { composeTestRule.onNodeWithText(okMessage).isDisplayed() }
+    composeTestRule.onNodeWithText(okMessage).performClick()
     composeTestRule.onNodeWithTag("TimeButton").performClick()
-    composeTestRule.waitUntil(15000) { composeTestRule.onNodeWithText("OK").isDisplayed() }
-    composeTestRule.onNodeWithText("OK").performClick()
+    composeTestRule.waitUntil(15000) { composeTestRule.onNodeWithText(okMessage).isDisplayed() }
+    composeTestRule.onNodeWithText(okMessage).performClick()
```

Comment:
Good job for using string defined in the ressource file for testing! It will help us to modify them without having to struggle to make the test pass.